src/locales/en/common.json

```diff
@@ -105,11 +105,29 @@
     },
     "ConnectAppTimeout": {
       "title": "Sorry, no device found",
-      "description": "Please make sure your {{productName}} is unlocked with Bluetooth enabled."
+      "description": {
+        "ios": {
+          "nanoS": "Please make sure your {{productName}} is unlocked and has Bluetooth enabled.",
```

Comment:
this case seems odd :sweat_smile: nanoS can't be connected via bluetooth. we can keep it

Reply:
there'd never be a case though, why keep the wording ? 🤔